index.html

```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="custom.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+    />
+    <title>Portfolio-Mobile Fiirst</title>
+  </head>
+  <body>
+    <div class="container">
```

Comment:
Kindly wrap your works section in a `<section>` tag🙏. This is for semantic reasons, as section tags are used to identify different parts of a webpage